### DISCUSSION TITLE
Windows are not automatically made the same size

### DISCUSSION THREAD

**Discussion by imac8t** [ayamir/nvimdots#1289] [Q&A] [answered]
ctrl-n打开关闭之后 分屏宽度要手动ctrl-w-=去恢复相等宽度

**Comment by Jint-lzxy** [comment, marked as answer]
I think this might be a duplicate of https://github.com/ayamir/nvimdots/discussions/1001. Does that fix your issue, or did you mean to restore the _relative_ window sizes? If it's the latter, you'll need an autocmd for that.
